dlp/snippets/inspect/inspect_test.go

```diff
@@ -276,3 +276,21 @@ func TestInspectBigquery(t *testing.T) {
 		})
 	}
 }
+
+func TestInspectWithCustomRegex(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := new(bytes.Buffer)
+
+	if err := inspectWithCustomRegex(buf, tc.ProjectID, "Patients MRN 444-5-22222", "[1-9]{3}-[1-9]{1}-[1-9]{5}", "C_MRN"); err != nil {
+		t.Errorf("inspectWithCustomRegex: %v", err)
```

Comment:
issue: replace `t.Errorf()` with `t.Fatal(err)`. If the test fails here, the next test will fail, too (most likely).

Reply:
fixed it.